medusa/server/core.py

```diff
@@ -23,22 +23,40 @@ def get_apiv2_handlers(base):
     from .api.v2.auth import AuthHandler
     from .api.v2.asset import AssetHandler
     from .api.v2.base import NotFoundHandler
+    from .api.v2.scene_exception import SceneExceptionHandler, SceneExceptionOperationHandler
 
     show_id = r'(?P<show_indexer>[a-z]+)(?P<show_id>\d+)'
     # This has to accept season of 1-4 as some seasons are years. For example Formula 1
-    ep_id = r'(?:(?:s(?P<season>\d{1,4})(?:e(?P<episode>\d{1,2}))?)|(?:e(?P<absolute_episode>\d{1,3}))|(?P<air_date>\d{4}\-\d{2}\-\d{2}))'
+    ep_id = r'(?:(?:s(?P<season>\d{1,4})(?:e(?P<episode>\d{1,2}))?)|(?:e(?P<absolute_episode>\d{1,3}))' \
+            r'|(?P<air_date>\d{4}\-\d{2}\-\d{2}))'
     query = r'(?P<query>[\w]+)'
     query_extended = r'(?P<query>[\w \(\)%]+)'  # This also accepts the space char, () and %
     log_level = r'(?P<log_level>[a-zA-Z]+)'
     asset_group = r'(?P<asset_group>[a-zA-Z0-9]+)'
 
     return [
-        (r'{base}/show(?:/{show_id}(?:/{ep_id})?(?:/{query})?)?/?'.format(base=base, show_id=show_id, ep_id=ep_id, query=query), ShowHandler),
-        (r'{base}/config(?:/{query})?/?'.format(base=base, query=query), ConfigHandler),
-        (r'{base}/log(?:/{log_level})?/?'.format(base=base, log_level=log_level), LogHandler),
-        (r'{base}/authenticate(/?)'.format(base=base), AuthHandler),
-        (r'{base}/asset(?:/{asset_group})(?:/{query})?/?'.format(base=base, asset_group=asset_group, query=query_extended), AssetHandler),
-        (r'{base}(/?.*)'.format(base=base), NotFoundHandler)
+        # All operations endpoints should be defined first.
+        (r'{base}/sceneexception/operation'.format(base=base),
```

Comment:
I need to think about this and I'll come back to you. Operations on a single resource or in a resource list. This was it's on a resource list